packages/gafl-webapp-service/src/handlers/__tests__/analytics-handler.spec.js

```diff
@@ -163,6 +163,36 @@ describe('The analytics handler', () => {
     })
   })
 
+  describe('checkAnalyticsCookiesPage', () => {
+    it.each([
+      [true, false, true, 'reject'],
+      [true, true, true, 'accept']
+    ])(
+      'sets selected to %s, acceptTracking to %s and seenMessage to %s when analyticsResponse is %s',
```

Comment:
Might be good to have a test to make sure nothing is updated if a value that isn't `accept` or `reject` is somehow submitted.